app/gui/src/main/scala/org/bitcoins/gui/GlobalData.scala

```diff
@@ -19,6 +19,17 @@ object GlobalData {
   val syncHeight: StringProperty = StringProperty("Syncing headers...")
 
   var network: BitcoinNetwork = _
+  val proxyEnabled = BooleanProperty(false)
+  val networkString: StringProperty = new StringProperty("")
+
+  def setBitcoinNetwork(
+      network: BitcoinNetwork,
+      proxyEnabled: Boolean): Unit = {
+    this.network = network
+    this.proxyEnabled.value = proxyEnabled
+    networkString.value = "Network: " + network
+    if (proxyEnabled) networkString.value += " over Tor"
```

Comment:
if we are using the bitcoind backend should we include `over Tor`?

Reply:
I was thinking from the point of view of informing the user that tor is running on the node connection. Whether "over Tor" or a pseudo indicator is the right way to do it, I'm not sure. I started thinking they were correlated, but I understand that's just the connection to the underlying node.

I would always show the state of tor just to acknowledge that it is running as the user set  with bitcoind backend as well as neutrino.

From my point of view, the user needs to know whether we are running a tor instance on their behalf. We could in the debug window, but I see it as an implementation detail they usually don't need to worry about.

Reply:
> From my point of view, the user needs to know whether we are running a tor instance on their behalf. 

I agree and this PR doesn't do that imo, it says their node is running over tor even when it might not be.